tests/summmarizer/test_image_description_retreiver.py

```diff
@@ -3,7 +3,7 @@
 
 import pytest
 
-from classifier_and_summarizer.summarizer.web_entity_detection import (
+from classifier_and_summarizer.summarization.web_entity_detection import (
```

Comment:
What about `classifier_and_summarizer` module?

Reply:
Will remove it in the next PR!